### ISSUE TITLE
Build failure

### ISSUE THREAD

**Issue by ChristopherA** [nhoogendoorn/CardStack#13]
On iOS 14.6 with Xcode 12.5 on M1 Mac, building the demo app CardStackExample.xcodeproject

Teaching myself SwiftUI, and it your UI looks like a good fit for a small app idea I have.

I'm getting two errors, both saying I need iOS 13 or better, but I am on 14+

```
/Users/christophera/Library/Developer/Xcode/DerivedData/CardStackExample-gpgukfuvsxjanihawijpiilcihwe/SourcePackages/checkouts/CardStack/Sources/CardStack/Protocols/CardData.swift:11:27: 'Identifiable' is only available in iOS 13 or newer

/Users/christophera/Library/Developer/Xcode/DerivedData/CardStackExample-gpgukfuvsxjanihawijpiilcihwe/SourcePackages/checkouts/CardStack/Sources/CardStack/Protocols/CardView.swift:11:27: 'View' is only available in iOS 13.0 or newer
```

**Comment by nhoogendoorn** [comment]
Hi @ChristopherA , just to double check: did you also set your `Deployment info` to have 13+?
You can find it in your project settings:

<img width="784" alt="Screenshot 2021-06-18 at 09 00 31" src="https://user-images.githubusercontent.com/29920922/122520334-c3a10d80-d013-11eb-82b5-1bf311b59337.png">

**Comment by ChristopherA** [comment]
Yes, it is checked.

![Card Stack Bug](https://user-images.githubusercontent.com/69103/130374534-a80ec73a-a19f-4bf9-b490-4e941d2fae5d.jpg)

**Comment by nhoogendoorn** [comment]
Thanks @ChristopherA ! I fixed it in the latest release `0.1.3`, let me know if everything works out!
